deps/v8/.gitignore

```diff
@@ -1,82 +0,0 @@
-*.a
```

Comment:
I think the less changes we have from upstream the better, even though in this case it has no impact on functionality, so I would say that we'd like to put it back in.